topics/icrc_27_accounts.md

```diff
@@ -101,8 +101,9 @@ None
 
 1. The relying party sends a `icrc27_accounts` request to the signer.
 2. Upon receiving the message, the signer first checks if it can process the message.
-    - If the relying party has not been granted the permission to invoke the method,
+    - If the relying party has been denied the permission to invoke the method,
       the signer sends a response with an error back to the relying party.
+      If the permission state is `ask_on_use`, the signer must prompt the user to either accept or deny this invocation.
```

Comment:
Even if it's kind of clear, maybe clarify what happens once the user accept or deny?

Reply:
Agreed, I added a reference to the relevant section of ICRC-25.